src/app/(routes)/shopping-mall/components/product-image/ProductImage.styled.ts

```diff
@@ -0,0 +1,11 @@
+import styled from "styled-components"
+
+export const RelativeContainer = styled.div`
+    position: relative;
+`
+
+export const AbsoluteContainer = styled.div`
+    position: absolute;
+    bottom: 0.75rem;
+    right: 0.35rem;
```

Comment:
Honestly, because it looks good. Since the price tag has quite a custom style (made with pure CSS with pseudoelements creating the triangle in the beginning and so one), the positioning needed ended up looking best with those two numbers.